node-stringprep.cc

```diff
@@ -19,13 +19,13 @@ class StringPrep : public ObjectWrap {
 public:
   static void Initialize(Handle<Object> target)
   {
-    NanScope();
-    Local<FunctionTemplate> t = NanNew<FunctionTemplate>(New);
+    Nan::HandleScope scope;
+    Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
     NanAssignPersistent(stringprep_constructor, t);
     t->InstanceTemplate()->SetInternalFieldCount(1);
-    NODE_SET_PROTOTYPE_METHOD(t, "prepare", Prepare);
+    Nan::SetPrototypeMethod(t, "prepare", Prepare);
 
-    target->Set(NanNew<String>("StringPrep"), t->GetFunction());
+    target->Set(Nan::New<String>("StringPrep").ToLocalChecked(), t->GetFunction());
```

Comment:
correct but you should use `Nan::Set(target, Nan::New<String>("StringPrep").ToLocalChecked(), t->GetFunction());` now for max-compatibility